dallinger/recruiters.py

```diff
@@ -77,6 +81,11 @@ class PsiTurkRecruiter(Recruiter):
 
     def __init__(self):
         """Set up the connection to MTurk and psiTurk web services."""
+
+        # commit to the db so if anything goes wrong the rest of the
+        # request is safe
+        session.commit()
```

Comment:
@thomasmorgan Okay, how does this look? I added commit statements in custom.py directly after the participant status is changed. The only place that I didn't add a commit statement was after setting the bonus, because I think the logic is a little different there, but not sure.

I'll test this out, but I think it's ready to merge. 

Just to double check, this latest version still achieves our aim of hiding the concept of database commits from experimenters, right?